Project2015To2017/Transforms/PropertySimplificationTransformation.cs

```diff
@@ -152,6 +152,9 @@ public void Transform(Project definition)
 					                         ):
 					case "RestorePackages" when ValidateDefaultValue("true"):
 					case "SchemaVersion" when ValidateDefaultValue("2.0"):
+					case "AssemblyVersion" when ValidateDefaultValue("1.0.0.0"):
```

Comment:
What if user specifies some version property to be `1.0`? Should we remove that too?

Reply:
Fixing this now, I must have imagined doing it earlier